api_tester/lib/api_tests/models/web_purchase_redemption_result_api_test.dart

```diff
@@ -14,20 +14,20 @@ class _WebPurchaseRedemptionResultApiTest {
       String obfuscatedEmail,
   ) {
     WebPurchaseRedemptionResult result =
-        WebPurchaseRedemptionResult.success(customerInfo: customerInfo);
-    result = WebPurchaseRedemptionResult.error(error: error);
-    result = const WebPurchaseRedemptionResult.purchaseBelongsToOtherUser();
-    result = const WebPurchaseRedemptionResult.invalidToken();
-    result = WebPurchaseRedemptionResult.expired(obfuscatedEmail: obfuscatedEmail);
+        WebRedemptionSuccess(customerInfo: customerInfo);
```

Comment:
This is the biggest breaking change. I wasn't sure how to namespace the results to WebPurchaseRedemptionResult, so I change the name of each value of the sealed class so it's at least more concrete... In any case, this was already a breaking change from #1365